src/components/chip/dropdown-chip-multi-select/index.tsx

```diff
@@ -101,7 +104,7 @@ export const DropdownChipMultiSelect = forwardRef<
                 leaveTo="scale-y-0 -translate-y-2/4"
               >
                 <Listbox.Options
-                  className={qtMerge(className, chipDropdownClassnames)}
+                  className={qtMerge(className, chipDropdownPanelClassnames)}
```

Comment:
@shafin-deriv, I'm not quite sure, but doesn't order matter here? 🤔 If it does, then I think you should put  `className` after `chipDropdownPanelClassnames`, as you'd want to overwrite the styles set by `chipDropdownPanelClassnames` with those set by `className`, right?

Reply:
Agreed! Done ✅